pandas/core/tools/numeric.py

```diff
@@ -96,7 +96,7 @@ def to_numeric(
         is to not use nullable data types. If specified, the behavior
         is as follows:
 
-        * ``"numpy_nullable"``: returns with nullable-dtype-backed
+        * ``"numpy_nullable"``: returns nullable-dtype-backed :class:`DataFrame`
         * ``"pyarrow"``: returns with pyarrow-backed nullable :class:`ArrowDtype`
 
```

Comment:
It looks to me like this function can return a Series, Index, scalar, or array (NumPy or pandas). Can either leave these lines as-is or say "object" instead of DataFrame.